### ISSUE TITLE
Regenerating the proto files make the tests fail

### ISSUE THREAD

**Issue by AdrienVannson** [danielgtaylor/python-betterproto#640]
### Summary

.

### Reproduction Steps

Re-generate the files used in the compiler, as documented in the `README`:

```
protoc \
    --plugin=protoc-gen-custom=src/betterproto/plugin/main.py \
    --custom_opt=INCLUDE_GOOGLE \
    --custom_out=src/betterproto/lib/std \
    -I /usr/include/ \
    /usr/include/google/protobuf/*.proto
```

Then, run `poe generate` and `poe test`

### Expected Results

The tests should pass.

### Actual Results

The `test_struct` test actually fails.

My guess is that the output files of the compiler have been manually edited to change the `to_dict` and `from_dict` methods of `Struct` (see https://github.com/danielgtaylor/python-betterproto/blame/master/src/betterproto/lib/std/google/protobuf/__init__.py#L2225 )

### System Information

libprotoc 3.12.4
Python 3.12.6
Name: betterproto
Version: 2.0.0b7
Summary: A better Protobuf / gRPC generator & library
Home-page: https://github.com/danielgtaylor/python-betterproto
Author: Daniel G. Taylor
Author-email: danielgtaylor@gmail.com
License: MIT
Location: /[...]/.venv/lib/python3.12/site-packages
Editable project location: [...]
Requires: grpclib, python-dateutil, typing-extensions
Required-by:

### Checklist

- [X] I have searched the issues for duplicates.
- [X] I have shown the entire traceback, if possible.
- [X] I have verified this issue occurs on the latest prelease of betterproto which can be installed using `pip install -U --pre betterproto`, if possible.

**Comment by Gobot1234** [comment]
Yeah that's just gonna need removing I think or we need a custom generator
